src/hpx_runtime.cpp

```diff
@@ -225,6 +225,11 @@ void hpx_runtime::barrier_wait(){
 bool hpx_runtime::start_taskgroup()
 {
     auto task = get_task_data();
+#if HPXMP_HAVE_OMP_50_ENABLED
+    kmp_taskgroup_t *tg_new = new kmp_taskgroup_t();
```

Comment:
Where is this memory being free'd?

Reply:
It's not freed, I will replace it with the intrusive pointer, thanks for pointing out!